### PR TITLE
DCOS-45308 - Muting flakes using xfailflake custom marker.

### PR DESCRIPTION
**Pull request by orsenthil**
## High-level description

* DCOS-45308 - Muting flakes using xfailflake custom marker.

### CODE REVIEW

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot label  ready for review

**Comment by orsenthil** [comment]
@adamtheturtle / @gpaul  / @jgehrcke  - Please review and comment on this PR and the EE PR.

File: packages/dcos-integration-test/extra/common.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import pytest
+
+xfailflake = pytest.mark.xfail(strict=False)
```
Comment (adamtheturtle) [inline packages/dcos-integration-test/extra/common.py]: Let's have a comment here with a description of why we have this and when to add / remove this marker.

Comment (jgehrcke) [reply packages/dcos-integration-test/extra/common.py]: Can do this in a follow-up though

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@adamtheturtle - I have addressed your review comment.

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot sync

**Comment by orsenthil** [comment]
The EE failures are the failures from parallel build experiment. This PR can be merged directly and merging this.

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Pull request merged**